tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -0,0 +1,140 @@
+#
+# Check for:
+#
+# ironicAPIReadyCount: 1
+# ironicConductorReadyCount: 1
+# apiEndpoints:
+#     ironic:
+#       admin: http://ironic-admin-openstack.apps-crc.testing
+#       internal: http://ironic-internal-openstack.apps-crc.testing
+#       public: http://ironic-public-openstack.apps-crc.testing
+# we should see and validate reason and status as below on all the DBReady, DBSyncReady, BootstrapReady, DeploymentReady, ExposeServiceReady, InputReady, IronicAPIReady, IronicConductorReady, IronicRabbitMqTransportURLReady, ServiceConfigReady etc.
+# reason: Ready
+#  status: "True"
+
+apiVersion: ironic.openstack.org/v1beta1
+kind: Ironic
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"ironic.openstack.org/v1beta1","kind":"Ironic","metadata":{"annotations":{},"name":"ironic","namespace":"openstack"},"spec":{"containerImage":"quay.io/openstack-k8s-operators/ironic-operator-index:latest","customServiceConfig":"[DEFAULT]\ndebug = true\n","databaseInstance":"openstack","databaseUser":"ironic","ironicAPI":{"containerImage":"quay.io/tripleozedcentos9/openstack-ironic-api:current-tripleo","replicas":1},"ironicConductor":{"containerImage":"quay.io/tripleozedcentos9/openstack-ironic-conductor:current-tripleo","pxeContainerImage":"quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo","replicas":1,"storageClass":"local-storage","storageRequest":"10G"},"ironicInspector":{"containerImage":"quay.io/tripleozedcentos9/openstack-ironic-inspector:current-tripleo","pxeContainerImage":"quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo","replicas":1},"secret":"osp-secret","serviceUser":"ironic","storageClass":"local-storage"}}
+  creationTimestamp: "2023-02-14T14:50:07Z"
```

Comment:
Fields like timestamps, and hashes will change between runs, so they should not be in the asserts

Reply:
done